src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller.Session_FilterModel.cs

```diff
@@ -457,7 +457,7 @@ public FilterResult(CompletionItem completionItem, string filterText, bool match
                 if (filterReason == CompletionFilterReason.BackspaceOrDelete &&
                     trigger.Kind == CompletionTriggerKind.Deletion)
                 {
-                    return item.FilterText.GetCaseInsensitivePrefixLength(filterText) > 0;
+                    return item.FilterText.GetCaseInsensitivePrefixLength(filterText) >= Math.Min(item.FilterText.Length, filterText.Length);
```

Comment:
How could it ever be greater than the min of those two?

Doesn't this change mean that it has to be identical to those?